### DISCUSSION TITLE
Password generation error?  cannot find module

### DISCUSSION THREAD

**Discussion by ghost** [wg-easy/wg-easy#1243] [General]
when I try to create the password hash using the documentation i get an error

root@wireguard:/home# docker run ghcr.io/wg-easy/wg-easy wgpw 'randompassword'
node:internal/modules/cjs/loader:1148
  throw err;
  ^

Error: Cannot find module '/app/wgpw'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.15.1

**Comment by eandersons** [comment]
As I have said before: just a little bit of search effort may go a long way: [Cannot find module '/app/wgpw'](/wg-easy/wg-easy/issues?q=is%3Aissue+Cannot+find+module+%27%2Fapp%2Fwgpw%27).

**Reply by ghost** [reply]
So does proper documentation

**Reply by eandersons** [reply]
The documentation is fine, the confusion comes from where to find it (development vs. production), and as a result the default branch contains changes that are not yet available in production. See https://github.com/wg-easy/wg-easy/issues/1197#issuecomment-2268213204 regarding this.

I agree that this should be communicated more clearly (this has also been mentioned on #1203).
